tests/user_restricted/test_user_restricted.py

```diff
@@ -57,16 +72,18 @@ def test_retrieve_patient_with_invalid_auth_header(self, headers):
         helpers.check_response_status_code(response, 401)
         helpers.check_response_headers(response, headers)
 
+    @pytest.mark.nhsd_apim_authorization(AUTH_HEALTHCARE_WORKER)
     def test_user_role_sharedflow_retrieve_patient_with_missing_urid_header(self, headers_with_token):
         self.headers.pop("NHSD-Session-URID")
         response = helpers.retrieve_patient(
-            retrieve[0]["patient"],
+            retrieve[10]["patient"],
             self.headers
         )
-        helpers.check_response_status_code(response, 200)
-        helpers.check_retrieve_response_body_shape(response)
+        helpers.check_retrieve_response_body(response, retrieve[10]["response"])
+        helpers.check_response_status_code(response, 400)
         helpers.check_response_headers(response, self.headers)
 
+    @pytest.mark.nhsd_apim_authorization(AUTH_HEALTHCARE_WORKER)
```

Comment:
Is there a more efficient way of marking these tests with the healthcare worker auth than adding them to each individually?

Reply:
Another one for [SPINEDEM-1680](https://nhsd-jira.digital.nhs.uk/browse/SPINEDEM-1680)!